flask_restplus/marshalling.py

```diff
@@ -49,9 +51,35 @@ def make(cls):
             out = OrderedDict([(envelope, out)])
         return out
 
-    items = ((k, marshal(data, v) if isinstance(v, dict)
-              else make(v).output(k, data))
-             for k, v in fields.items())
+    items = []
+    keys = []
+    for dkey, val in fields.items():
+        key = dkey
+        if isinstance(val, dict):
+            value = marshal(data, val)
+        else:
+            field = make(val)
+            # exclude already parsed keys from the wildcard
+            if isinstance(field, Wildcard) and keys:
+                for tmp in keys:
+                    if tmp not in field.exclude:
+                        field.exclude.append(tmp)
+                keys = []
+            value = field.output(dkey, data)
+            if isinstance(field, Wildcard):
+                key = field.key or dkey
+                items.append((key, value))
+                while True:
+                    value = field.output(dkey, data)
+                    if value is None or value == field.default:
+                        break
+                    key = field.key
+                    items.append((key, value))
+                continue
+        keys.append(key)
+        items.append((key, value))
+
+    items = tuple(items)
```

Comment:
I understand the issue here, but the *Wildcard* thing will require deep introspection and thus, people can/may/should expect slow results.